DataTable.java

```diff
@@ -2,7 +2,30 @@
 
 public class DataTable {
     public static void showData() {
-        // i have to move it over to database connection
+        CommandLineTable dataTable = new CommandLineTable();
+        String showDataMenu = "\n *** SHOW DATA ***\n\n" + "1. Daily weight\n" + "2. Weekly weight\n"
+                + "\nEnter your choice: ";
+        System.out.print(showDataMenu);
+        char choice = main.keyboard.nextLine().charAt(0);
+        System.out.println("--------------");
+        switch (choice) {
+            case '1':
+                dataTable = getDailyWeightTable();
+                System.out.println("Daily Weight Table:");
+                break;
+            case '2':
+                dataTable = getWeeklyWeightTable();
+                System.out.println("Weekly Weight Table:");
+                break;
+            default:
+                dataTable = null;
+                System.out.println("Choice not implemented!");
+                break;
+        }
+        dataTable.print();
```

Comment:
again, you should add at least another line break to make it more visually pleasing...